tests/input/examples/schema_definition-native-array-2.yaml

```diff
@@ -0,0 +1,127 @@
+id: https://example.org/arrays
+name: arrays-temperature-example-2
+title: Array Temperature Example Using NDArray Classes
+description: |-
+  Example LinkML schema to demonstrate a 3D DataArray of temperature values with labeled axes
+  using LinkML NDArray classes
+license: MIT
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+  wgs84: http://www.w3.org/2003/01/geo/wgs84_pos#
+  example: https://example.org/
+
+default_prefix: example
+
+imports:
+  - linkml:types
+
+classes:
+
+  TemperatureDataset:
+    tree_root: true
+    implements:
+      - linkml:DataArray
+    annotations:
+      array_data_mapping:
+        data: temperatures_in_K
+        dims: [x, "y", t]
+        coords:
+          latitude_in_deg: x
+          longitude_in_deg: "y"
+          time_in_d: t
+    attributes:
+      name:
+        identifier: true
+        range: string
+      latitude_in_deg:
+        implements:
+          - linkml:axis
+        range: LatitudeSeries
```

Comment:
# NOTE: `LatitudeSeries` could have multiple array attributes. Use the one that implements `linkml:elements`. We should also ensure that exactly one attribute implements `linkml:elements`.